src/backend/distributed/shared_library_init.c

```diff
@@ -585,6 +585,19 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_local_reference_table_foreign_keys",
+		gettext_noop("Enables foreign keys from/to local tables"),
+		gettext_noop("When enabled any foreign keys between local tables and reference "
+					 "tables supported by converting the local table to a citus local "
```

Comment:
Small typo:
When enabled, foreign keys between local tables and references tables are supported by converting the local table to a citus local table.

(although I'm not sure whether we should mention citus local table a lot)

Reply:
avoided "citus local table"